common/Views/AdaptiveCardViews/EnvironmentsSettingsCard.xaml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<ctControls:SettingsCard
+    x:Class="DevHome.Common.Views.AdaptiveCardViews.EnvironmentsSettingsCard"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:ctControls="using:CommunityToolkit.WinUI.Controls"
+    mc:Ignorable="d">
+
+    <!-- 
+        Override SettingsCard default resources. This is used because the default
+        SettingsCardHeaderIconMaxSize value for settings cards is 20. This makes
+        the icons sent from extensions in the creation flow appear too small and pixelated.
+        So we override it here, to make sure they are bigger.
+    -->
+    <ctControls:SettingsCard.Resources>
+        <x:Double x:Key="SettingsCardHeaderIconMaxSize">40</x:Double>

```

Comment:
Hmm I don't believe it can be done directly to the settings card resource, but thinking about it now. I think if I add 

`<x:Double x:Key="SettingsCardHeaderIconMaxSize">40</x:Double>`

into the resources of the xaml control I'm using the settings card in, that should override references to `SettingsCardHeaderIconMaxSize` within it. Just like here: https://github.com/microsoft/devhome/blob/5d6429dd5f1c639243252096092edd8342f77fa7/tools/SetupFlow/DevHome.SetupFlow/Views/MainPageView.xaml#L21. 

I'll update this thanks